### ISSUE TITLE
 TypeError: create_autogen_memgpt_agent() got an unexpected keyword argument 'persona_description'

### ISSUE THREAD

**Issue by hherpa** [cpacker/MemGPT#504]
Error "TypeError: create_autogen_memgpt_agent() got an unexpected keyword argument 'persona_description'" when launching the official memgpt google coollaboratory notebook : "[memgpt_coder_autogen.ipynb](https://github.com/cpacker/MemGPT/blob/main/memgpt/autogen/examples/memgpt_coder_autogen.ipynb)"

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
[<ipython-input-7-396950d9e99f>](https://localhost:8080/#) in <cell line: 47>()
     55     # In our example, we swap this AutoGen agent with a MemGPT agent
     56     # This MemGPT agent will have all the benefits of MemGPT, ie persistent memory, etc.
---> 57     coder = create_autogen_memgpt_agent(
     58         "MemGPT_coder",
     59         persona_description="I am a 10x engineer, trained in Python. I was the first engineer at Uber (which I make sure to tell everyone I work with).",

TypeError: create_autogen_memgpt_agent() got an unexpected keyword argument 'persona_description'
```
I didn't edit the code in any way (except that I inserted my openal API key), but your notepad still didn't start :(

**Comment by caudurodev** [comment]
Same issue here. I think the arguments for create_autogen_memgpt_agent have changed completely. Are there any docs?

This is what I'm getting with the latest pip install:
```
create_autogen_memgpt_agent(agent_config, interface=None, interface_kwargs={}, persistence_manager=None, persistence_manager_kwargs=None, default_auto_reply: Union[str, Dict, NoneType] = '', is_termination_msg: Optional[Callable[[Dict], bool]] = None)
    See AutoGenInterface.__init__ for available options you can pass into
    `interface_kwargs`.  For example, MemGPT's inner monologue and functions are
    off by default so that they are not visible to the other agents. You can
    turn these on by passing in
    ```
    interface_kwargs={
        "debug": True,  # to see all MemGPT activity
        "show_inner_thoughts: True  # to print MemGPT inner thoughts "globally"
                                    # (visible to all AutoGen agents)
    }
```

**Comment by simonngai** [comment]
> Same issue here. I think the arguments for create_autogen_memgpt_agent have changed completely. Are there any docs?
> 
> This is what I'm getting with the latest pip install:
> 
> ```
> create_autogen_memgpt_agent(agent_config, interface=None, interface_kwargs={}, persistence_manager=None, persistence_manager_kwargs=None, default_auto_reply: Union[str, Dict, NoneType] = '', is_termination_msg: Optional[Callable[[Dict], bool]] = None)
>     See AutoGenInterface.__init__ for available options you can pass into
>     `interface_kwargs`.  For example, MemGPT's inner monologue and functions are
>     off by default so that they are not visible to the other agents. You can
>     turn these on by passing in
>     ```
>     interface_kwargs={
>         "debug": True,  # to see all MemGPT activity
>         "show_inner_thoughts: True  # to print MemGPT inner thoughts "globally"
>                                     # (visible to all AutoGen agents)
>     }
> ```

I think so. In 0.2.3, the definition of  create_autogen_memgpt_agent expects a name not a agent_config, but I have no idea how to pass the agent_config.

**Comment by sarahwooders** [comment]
Can you share what version of MemGPT you are running? You can find it with `memgpt version`.

**Comment by hherpa** [comment]
> Can you share what version of MemGPT you are running? You can find it with `memgpt version`.

@sarahwooders 0.1.16 , but I have a new version now and the error is for sure

**Comment by cpacker** [comment]
Should be patched in the latest version (0.2.5 coming out tonight, or try `pip install pymemgpt-nightly`): https://memgpt.readthedocs.io/en/latest/autogen/

Please reopen if this is still not working for you (or ping us on Discord #support)

**Comment by igoventura** [comment]
@simonngai I'm passing the `agent_config` as follows:

```python
from memgpt.config import AgentConfig
from dotenv import load_dotenv

load_dotenv()

agent_config = AgentConfig(
        "I am a 10x engineer, trained in Python. I was the first engineer at Uber which I make sure to tell everyone I work with).",
        f"You are participating in a group chat with a user ({user_proxy.name}) and a product manager ({pm.name}).",
        os.getenv("model"),
        name="MemGPT_coder"
    )
```

And also I need to set the following env on my dotenv file:

`MEMGPT_CONFIG_PATH=MEMGPT_CONFIG`

And then, finally, create a file called MEMGPT_CONFIG with the following values:

```
[openai]
key=sk-YOUROAIKEY

[model]
model_endpoint=https://api.openai.com/v1
model_endpoint_type=openai
```
